ffi/derive/src/wrapper.rs

```diff
@@ -29,36 +117,27 @@ fn gen_shared_fns(name: &Ident, attrs: &[syn::Attribute]) -> Vec<TokenStream> {
         }
 
         if let syn::Meta::List(derives) = attr.parse_meta().expect("Derive macro invalid") {
-            let mut valid_derives = vec![];
-
-            let mut derive_eq = false;
-            let mut derive_ord = false;
             for derive in derives.nested {
                 if let syn::NestedMeta::Meta(meta) = &derive {
                     if let syn::Meta::Path(path) = meta {
-                        if path.is_ident("Clone") {
-                            shared_fn_impls.push(quote! {
-                                impl Clone for #name {
-                                    fn clone(&self) -> Self {
-                                        let mut output = core::mem::MaybeUninit::uninit();
-
-                                        let handle_id = iroha_ffi::FfiConvert::into_ffi(<#name as iroha_ffi::Handle>::ID, &mut ());
-                                        let clone_result = unsafe { crate::__clone(handle_id, self.__opaque_ptr, output.as_mut_ptr()) };
-
-                                        if clone_result != iroha_ffi::FfiReturn::Ok  {
-                                            panic!("Clone returned: {}", clone_result);
-                                        }
-
-                                        unsafe {iroha_ffi::FfiOutPtrRead::try_read_out(output.assume_init()).expect("Invalid output")}
-                                    }
-                                }
-                            });
+                        if path.is_ident("Copy") {
```

Comment:
You cannot have both `Copy` and `Drop` implemented for the same type. I was thinking to add a comment but since this wouldn't compile if it is changed it would be obvious right away why it is so